prover/src/pp.rs

```diff
@@ -28,28 +31,31 @@ where
     Ok(())
 }
 
-pub fn load_pp<SP>(file: &str) -> Result<PP<SP>, ProofError>
+pub fn load_pp<C, SP>(file: &str) -> Result<PP<C, SP>, ProofError>
 where
+    C: CommitmentScheme<P1>,
     SC: StepCircuit<F1> + Sync,
-    SP: SetupParams<G1, G2, C1, C2, RO, SC> + Sync,
+    SP: SetupParams<G1, G2, C, C2, RO, SC> + Sync,
```

Comment:
We should keep `C1 C2` notation

Reply:
C1 is a type alias for Pedersen<G1>: here, we want C to be generic so it works
for PVC as well.

Reply:
Didn't notice it because of wildcard imports unfortunately